src/plugins/codeStyle/index.ts

```diff
@@ -125,6 +137,12 @@ export default class CodeStyle {
                     diagnostics.push(messages.noPrint(s.tokens.print.range, noPrint));
                 }
             },
+            LiteralExpression: e => {
```

Comment:
Can you add another handler for TemplateStringExpression if they're not already handled by this?

```
someColor = `#00FF00`
```

Reply:
Done